<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugWindow</class>
 <widget class="QWidget" name="DebugWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debug</string>
  </property>
  <widget class="QSpinBox" name="SBItemNum">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::StrongFocus</enum>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnExitDebug">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit Debug Mode</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CBGoldenFinger">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>GoldenFinger    set item to:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CBEnabled">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Debug Mode Enabled</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
